web/templates/message_controls.hbs

```diff
@@ -1,36 +1,19 @@
-<div class="message_controls no-select">
```

Comment:
Does removing `no-select` here mean that copy/pasting a message will now include the controls icons? Worth checking carefully with both forms of select (on Linux, highlighting text with the mouse and then pasting with the middle button and Ctrl+C are independent copy/paste mechanisms).